.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: groovy
+jdk:
+- oraclejdk8
```

Comment:
Travis CI is using 1.8.0_65 as is... that doesn't seem so bad. In the interest of saving build time, it seems best not do `sudo apt-get update && sudo apt-get install oracle-java8-installer` as your junit example does. What do you think?